packages/inventory-compliance/src/ComplianceRemediationButton.js

```diff
@@ -30,7 +30,11 @@ class ComplianceRemediationButton extends React.Component {
     )
 
     rulesWithRemediations = (rules, system) => {
-        return rules.filter(rule => rule.remediationAvailable).map(
+        return rules.filter((rule) => (
+            rule.remediationAvailable &&
+            this.ruleProfile(rule, system).supported &&
```

Comment:
Observation: I've checked out `ruleProfile()` and it pulls in a profile by referencing `ref_id` of a rule and its profile. That seems insufficient and could uncover some bugs. UUID from `id` field would be preferred.